docs/source/fundamentals/tips.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Tips for success
+description: Your starting point for learning GraphQL and Apollo
+---
+
+TK WIP TODO finish this intro.  Depending on the size of an organization, there are usually existing practices surrounding the management of the data 
+
+<h2 id="incremental-adoption">Incremental adoption</h2>
+
+While it makes an excellent longer-term goal, GraphQL doesn’t need to stand above the entire data tree in an organization in order to provide value and expending large amounts of engineering effort to switch an entire stack should generally be avoided.
+
+Unlike some technologies, which are difficult to embrace without getting buy-in from all the teams in an organization, GraphQL can be utilized by smaller units (e.g. component by component, team by team) and later integrated into the bigger picture.
+
+Thanks to GraphQL’s ability to fetch data from any data source, it makes sense to keep existing APIs as they are and use GraphQL to “wrap” the data.  This can be appreciated by organizations with existing collections of REST (or similar) APIs which power their user experiences since they can lean heavily on their existing investments.  This allows small teams to immediately realize many benefits of GraphQL without waiting for deeper adoption and allows existing versions of the application, which have already been deployed and rely on the existing API, to keep functioning uninterrupted.
+
```

Comment:
I think we need a little more information on the how here. First, teams should start building their schema with one feature. They should develop the ideal API for that feature based on how they're consuming it on the client. Then, in their resolvers, they can call the resource associated with that feature + perform any data transformation that used to exist in Redux action creators. From there, they can start to integrate Apollo Client with the front-end code for that feature.